.gitignore

```diff
@@ -4,6 +4,7 @@
 *.dll
 *.so
 *.dylib
+manifests/
```

Comment:
Note that this will also ignore code changes in pkg/manifests/. I think need to use _./manifests_ here or *.yaml to just ignore the manifest files.

Reply:
This doesn't ignore the changes done in `pkg/manifests/clusterdeployment.go` or ` pkg/manifests`

Reply:
You're right, I do see changes made to *pkg/manifests/* files . However I don't see new files added there for some reason. For example, with this change I see:
```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   pkg/manifests/clusterdeployment.go
	modified:   pkg/manifests/infraenv.go

no changes added to commit (use "git add" and/or "git commit -a")
```
While a new file that was added to *pkg/manifests/* only shows if I remove this:
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   .gitignore
	modified:   pkg/manifests/clusterdeployment.go
	modified:   pkg/manifests/infraenv.go

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	pkg/manifests/manifests.go

Reply:
The ignore rules only apply to "new" files not already being tracked as part of the repo history.

Reply:
Yeah, it will ignore new files. I had the same problem with setting `bin` ignoring `/data/ignition/files/usr/local/bin`. Putting `/bin` instead did the Right Thing, so I think this needs to be `/manifests`.

Reply:
Yeah it looks like ` /manifests` works. I see the new file in` pkg/manifests/` but not in `./manifests`

Reply:
Done